web/client/components/catalog/RecordItem.jsx

```diff
@@ -218,6 +218,8 @@ class RecordItem extends React.Component {
                 caption={
                     <div>
                         {!this.props.hideIdentifier && <div className="identifier">{record && record.identifier}</div>}
+                        {record?.serviceType === '3dtiles' && record?.catalogType === 'csw' && <small className="text-info">
```

Comment:
I'm not sure about this change, since I cannot test it and see how it looks like ? 

Can you provide a screenshot about it ?

Reply:
![image](https://github.com/geosolutions-it/MapStore2/assets/58145645/1d73f2ea-0764-45f3-8860-c656c8011fc9)
@MV88

Reply:
looks good, next time do not highlight it with the yellow otherwise you will not get clearly the final result